test/.eslintrc

```diff
@@ -10,6 +10,10 @@
         "../.eslintrc",
         "plugin:mocha/recommended"
     ],
+    "overrides": [{
+        "files": ["**/*.ts"],
+        "extends": ["../src/.eslintrc.js"]
```

Comment:
I don't know much about this but is this overrides the `extends` defined just above?

Reply:
this defines separate config for files under this directory matching the pattern. we use the config in src for typescript files for them.

Reply:
which means it overrides the config of certain files with different config

Reply:
But we have some config options defined in the top level .eslintrc file that are not there in src/.eslintrc.js, so I feel like we change something (but not sure :smile: as I don't know eslint much).

BTW, I am really confused by the amount of different .eslint config files we have :smile: Ideally, I guess we just need to have one defined in the top level, shouldn't we?